downloads.html

```diff
@@ -102,6 +102,34 @@ <h3 class="download-version">2.7.0<a href="#2.7.0"><i class="fas fa-link " style
         For more information, please read the detailed <a href="https://downloads.apache.org/kafka/2.7.0/RELEASE_NOTES.html">Release Notes</a>.
     </p>
 
+      <span id="2.6.2"></span>
+      <h3 class="download-version">2.6.2<a href="#2.6.2"><i class="fas fa-link " style="color:#053ce2"></i></a></h3>
+      <ul>
+          <li>
+              Released April 20, 2021
+          </li>
+          <li>
+              <a href="https://downloads.apache.org/kafka/2.6.2/RELEASE_NOTES.html">Release Notes</a>
+          </li>
+          <li>
+              Source download: <a href="https://downloads.apache.org/kafka/2.6.2/kafka-2.6.2-src.tgz">kafka-2.6.2-src.tgz</a> (<a href="https://downloads.apache.org/kafka/2.6.2/kafka-2.6.2-src.tgz.asc">asc</a>, <a href="https://downloads.apache.org/kafka/2.6.2/kafka-2.6.2-src.tgz.sha512">sha512</a>)
+          </li>
+          <li>
+              Binary downloads:
+              <ul>
+                  <li>Scala 2.12 &nbsp;- <a href="https://downloads.apache.org/kafka/2.6.2/kafka_2.12-2.6.2.tgz">kafka_2.12-2.6.2.tgz</a> (<a href="https://downloads.apache.org/kafka/2.6.2/kafka_2.12-2.6.2.tgz.asc">asc</a>, <a href="https://downloads.apache.org/kafka/2.6.2/kafka_2.12-2.6.2.tgz.sha512">sha512</a>)</li>
```

Comment:
I found kind of a weird mix of different urls across the various releases on the downloads page (eg `https://www.apache.org/dyn/closer.cgi`), but presumably we want to point to `https://downloads.apache.org/kafka/2.6.2`?

Reply:
Actually, we want to follow that weird convention. I think the SHA512 and ASC files are small enough that we just use the downloads, but the other archive files are large enough that we should the `https://www.apache.org/dyn/closer.cgi` mechanism, which chooses a mirror that is closest to the user.

So let's change the links to be similar to the other recent releases.